client/app/app.html

```diff
@@ -3,3 +3,78 @@
 <a ui-sref="orderbook">Orderbook</a> | <a ui-sref="chart">Chart</a>
 <div ui-view>
 </div>
+
+	<div class="tx" ng-controller="orderbookCtrl" ng-model="testData">
+		      <h4>Exchange order book</h4> 
+	        <div class="order">Bid</div>
+	        <div ng-repeat="d in testData.bids | limitTo: 20">
+	         	<h7> {{ d[0] }} </h7>
+	         	<h7 style="margin-left: 40px"> {{ d[1] }} </h7>
+	         	<h7 style="float: right"> {{ d[2] }} </h7>
+	         		        </div>
+	        <div class="close"></div>
+	        <div class="order">Ask</div>
+ 					<div ng-repeat="d in testData.asks | limitTo: 20">
+	         	<h7> {{ d[0] }} </h7>
+	         	<h7 style="margin-left: 40px"> {{ d[1] }} </h7>
+	         	<h7 style="float: right"> {{ d[2] }} </h7>
+	         	</div>
+	    </div>
+	 </div>
+
+<!--   <div class="chart" ng-controller="chartCtrl" ng-model="">
+    <chartist class="ct-chart ct-golden-section" chartist-data="Chartist.barData" chartist-chart-type="Bar" chartist-chart-options="Chartist.barOptions" chartist-responsive-options="Chartist.barResponsiveOptions">
+      
+    </chartist>
+  </div> -->
+
+	<div style="float:right" class="execution" ng-controller="executionCtrl" >
+ 		<div class="close"> <h4>Place Orders</h4></div>
+    <div class="order" style="position: absolute"> <h3> Order Type </h3></div>
+    <select style="margin-left: 120px; margin-top: 5px"> 
+    	<option value="market">Market</option>
+			<option value="limit">Limit</option>
+    </select>
+		<div style="margin-top: 30px">
+   		Lot Size<input style="margin-left: 15px" id="size" type="number" value="1" min="0.0001" max="100" ng-model="order.size" /><br>
+    </div>
+    <div style="margin-top: 30px">
+   	  Price<input style="margin-left: 15px" id="price" type="number" min="" max="" ng-model="order.price" /><br>
+    </div>
+    <div style="margin-top: 20px">
+      <button id="buy" ng-click="buy()">Buy</button>
+      <button id="sell" ng-click="sell()">Sell</button>
+    </div>
+  </div>
+
+  <div style="float:right" class="wallet" ng-controller="walletCtrl">
+    <div class="close"> <h4>Wallet Address:</h4>
+        <h4> {{ showWallet }}</h4>
+    </div>
+    <div style="position: absolute"> <h3></h3></div>
+    <div style="margin-top: 30px">
+      Balance
+    </div>
+    <div style="margin-top: 20px">
```

Comment:
All of the inline styling in this file needs to go! 
It should all be handled through the CSS.